sdk/test/test_helper.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+# Copyright 2019 OpenTelemetry Authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+require 'simplecov'
+SimpleCov.start
+SimpleCov.minimum_coverage 85
+
+require 'opentelemetry'
```

Comment:
Should be `opentelemetry/sdk`.

This is going to be a merge conflict with #59 but I'll deal with that.

Reply:
whoops I thought I added that.